appveyor.yml

```diff
@@ -16,31 +19,51 @@ skip_commits:
     message: /\[ci skip\]/
 
 cache:
-    - c:\php -> appveyor.yml
+    - C:\ProgramData\chocolatey\bin -> appveyor.yml
+    - C:\ProgramData\chocolatey\lib -> appveyor.yml
+    - C:\tools\php5.6 -> appveyor.yml
```

Comment:
If you make the version a subdirectory of "C:\tools\php", this and other occurrences may not turn that ugly. :)